ja2003.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.christianrauch.Jahresarbeit-2003</id>
+  
+  <name>Jahresarbeit 2003</name>
+  <summary>Find die größte Fläche von zusammenhängenden Quadraten</summary>
+
+  <url type="homepage">https://github.com/christianrauch/Jahresarbeit-2003</url>
+  
+  <project_license>GPL-2.0 AND OFL-1.1 AND Apache-2.0</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  
+  <recommends>
+    <display_length compare="ge">1024</display_length>
+  </recommends>
+  
+  <description>
+    <p>
+      Das Programm &quot;Jahresarbeit 2003&quot; entstand als Jahresarbeit im Fach Informatik. Nach Spielstart sieht man ein 9x9 Feld mit verschiedenfarbigen Quadraten. Am Rand des Feldes sind Pfeile zu erkennen, welche genutzt werden um die Reihen und Spalten in denen sich die Quadrate befinden zu verschieben. Zusammenhängende gleichfarbige Quadrate bilden eine Fläche, welche durch draufklicken entfernt wird. Ziel des Spieles ist es eine möglichst große Fläche in kürzester Zeit weg zu klicken. Das Spiel ist zu Ende wenn alle Quadrate entfernt sind oder es keine Möglichkeit mehr gibt um die restlichen Quadrate zu entfernen. Jedoch sollte man um alle Quadrate zu entfernen so wenig Klicks wie möglich gebrauchen, da diese die Endpunktzahl verringern.
+    </p>
+  </description>
+
+  <content_rating type="oars-1.0" />
+
+  <releases>
+    <release date="2021-12-26" version="2.0"/>
+  </releases>
+  
+  <launchable type="desktop-id">com.github.christianrauch.Jahresarbeit-2003.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>Hauptmenü</caption>
+      <image>https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_800,h_600/https://dashboard.snapcraft.io/site_media/appmedia/2019/07/menu.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Spiel</caption>
+      <image>https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_800,h_600/https://dashboard.snapcraft.io/site_media/appmedia/2019/07/game.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Spiel Beschreibung</caption>
+      <image>https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_800,h_600/https://dashboard.snapcraft.io/site_media/appmedia/2019/07/info.png</image>
```

Comment:
I think it would be nice to host these screenshots elsewhere (even directly on GitHub) instead.

Reply:
That means I have to create a new repo or add them to the code repo. I may do this in the future by adding a README with the screenshots to the upstream repo, but for now, I don't see the advantage of this extra work. Hosting them on https://dashboard.snapcraft.io will be as stable as any other cloud service.